src/api/app/controllers/application_controller.rb

```diff
@@ -456,4 +457,14 @@ def forward_from_backend(path)
   def shutup_rails
     Rails.cache.silence! unless Rails.env.development?
   end
+
+  def set_influxdb_tags
+    anonymous = !User.current || (User.current && User.current.is_nobody?)
```

Comment:
[nitpick] perhaps this is easier on the eyes:
```
anonymous = User.current_login == User::NOBODY_LOGIN

Reply:
Hm, we still need ``User.current || `` as it seems there are cases when User.current is nil.

Reply:
yes, but User.current_login handles that